metadata-ingestion/src/datahub/ingestion/source/sql/presto_on_hive.py

```diff
@@ -488,22 +527,18 @@ def loop_tables(
             dataset_snapshot.aspects.append(schema_metadata)
 
             # add table properties
-            default_properties = ["create_date", "table_type", "table_location"]
-            properties: Dict[str, str] = {}
-            for prop in default_properties + self.config.extra_properties:
-                if prop in columns[-1]:
-                    properties[prop] = str(columns[-1][prop]) or ""
+            properties: Dict[str, str] = properties_cache.get(dataset_name, {})
+            properties["table_type"] = str(columns[-1]["table_type"]) or ""
+            properties["table_location"] = str(columns[-1]["table_location"]) or ""
+            properties["create_date"] = str(columns[-1]["create_date"]) or ""
 
             par_columns: str = ", ".join(
                 [c["col_name"] for c in columns if c["is_partition_col"]]
             )
             if par_columns != "":
                 properties["partitioned_columns"] = par_columns
 
-            table_description = (
-                columns[-1]["description"] if "description" in columns[-1] else ""
-            )
-
+            table_description = properties.get("comment", "")
```

Comment:
```suggestion
            table_description = properties.get("comment")
```
If "comment" property is absent in properties, maybe we should not set table_description to "" , and instead keep it null.